src/ice_match/rule_0/main.py

```
@@ -19,133 +19,133 @@
-        
+
         Args:
             filepath: Path to CSV file
             source: Trade source (TRADER or EXCHANGE)
-            
+
         Returns:
             List of trades
         """
         if not filepath.exists():
             raise FileNotFoundError(f"File not found: {filepath}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)